cmd/opencensusd/main.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Program opencensusd collects OpenCensus stats and traces
+// to export to a configured backend.
+package main
```

Comment:
Can we rename this repo to "census-instrumentation/opencensusd"?

Reply:
opencensusd will only be one of the binaries/components from the service. We are planning to add more in the future. That's why the repo is named opencensus-service.